src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/FallbackRazorConfiguration.cs

```diff
@@ -40,7 +40,11 @@ internal class FallbackRazorConfiguration : RazorConfiguration
          "MVC-3.0",
          new[] { new FallbackRazorExtension("MVC-3.0"), });
 
-    public static readonly RazorConfiguration Latest = MVC_3_0;
+    public static readonly RazorConfiguration Latest = new FallbackRazorConfiguration(
```

Comment:
Does anyone know how "MVC versions" are expected to map to "Razor language versions"?

I can't tell if they are separate concepts, or this file is just old, and missing everything between 5 and 8. The MVC versions that _are_ in this file seem to line up with RazorLanguageVersion entries.

@danroth27? @NTaylorMullen?